cert-manager/base/certificate.yaml

```diff
@@ -8,6 +8,6 @@ spec:
   issuerRef:
     name: cert-manager-acme-dns-issuer
     kind: ClusterIssuer
-  commonName: $(certManagerCommonName)
+  commonName: '$(certManagerCommonName)'
```

Comment:
@rushtehrani Why do we need the single quotes here? Doesn't the result generate double quotes?

Reply:
I can answer that. If he doesn't add quotes, wildcard doesn't work.

Reply:
See: https://onepanelio.slack.com/archives/G9HV7FU9G/p1583545162266700